src/components/FormContainer.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+
+const FormContainer = props => <div className="container Container-bg"><h3>{props.title}</h3>{props.children}</div>
```

Comment:
please format this. I also think that maybe this would throw an error?

Reply:
formatted FormContainer logic

Reply:
Verified changes made